ena-submission/src/ena_deposition/create_project.py

```diff
@@ -88,13 +88,81 @@ def construct_project_set_object(
     return ProjectSet(project=[project_type])
 
 
-def submission_table_start(db_config: SimpleConnectionPool):
+def set_project_table_entry(db_config, config, row):
```

Comment:
This function would be a logical place to test for existence of the bioproject and error if it doesn't exist